### ISSUE TITLE
Wrong object IDs in Entity relationship table

### ISSUE THREAD

**Issue by timrobertson100** [gbif/model-tests#8]
```
arctos=# ALTER TABLE entity_relationship ADD FOREIGN KEY ("objectEntityID") REFERENCES entity_of_interest ("entityID");
ERROR:  foreign key constraint "entity_relationship_objectEntityID_fkey" cannot be implemented
DETAIL:  Key columns "objectEntityID" and "entityID" are of incompatible types: character varying and numeric.
```

**Comment by timrobertson100** [comment]
This seems to be caused by using both local and external IDs in the `objectEntityID` in `entity_relationship`

```
arctos=# select * from entity_relationship;
 entityRelationshipID | dependsOnEntityRelationshipID | subjectEntityID | entityRelationshipType |                   objectEntityID                   
----------------------+-------------------------------+-----------------+------------------------+----------------------------------------------------
                      |                               |        21714980 | host of                | http://arctos.database.museum/guid/DMNS:Para:664
                      |                               |        21714980 | host of                | http://arctos.database.museum/guid/DMNS:Para:64
                      |                               |        21714980 | host of                | http://arctos.database.museum/guid/DMNS:Para:65
                      |                               |        29610849 | parasite of            | http://arctos.database.museum/guid/DMNS:Mamm:11098
                      |                               |        29610849 | collected with         | http://arctos.database.museum/guid/DMNS:Para:664
                      |                               |        29610849 | collected with         | http://arctos.database.museum/guid/DMNS:Para:65
                      |                               |        29610851 | parasite of            | http://arctos.database.museum/guid/DMNS:Mamm:11098
                      |                               |        29610851 | collected with         | http://arctos.database.museum/guid/DMNS:Para:664
                      |                               |        29610851 | collected with         | http://arctos.database.museum/guid/DMNS:Para:64
                      |                               |        10007944 | shows cataloged_item   | 10599395
                      |                               |        10007945 | shows cataloged_item   | 10599395
                      |                               |        10203450 | shows cataloged_item   | 10599395
                      |                               |        10212653 | shows cataloged_item   | 10599395
...
...
...
```

**Comment by timrobertson100** [comment]
Looking at the EOI identifier table, I think some can be mapped:

| **current** | **proposed**                                      |
|----------|-------------------------------------------------------|
| http://arctos.database.museum/guid/DMNS:Para:664  | _not found_                                                       |
| http://arctos.database.museum/guid/DMNS:Para:64  | 29610849                                                       |
| http://arctos.database.museum/guid/DMNS:Para:65  | 29610851                                                       |
| http://arctos.database.museum/guid/DMNS:Mamm:11098  | 21714980 perhaps?                                                       |
```

**Comment by timrobertson100** [comment]
Closed as we'll discuss this on #9 instead

**Comment by tucotuco** [comment]
My first quick guess here without looking is that the numbers you are
trying to match are occurrenceIDs, whereas the EOI Ids from Arctos really
are organismIDs.
